sechub-integrationtest/src/main/java/com/mercedesbenz/sechub/integrationtest/api/AssertUserJobInfo.java

```diff
@@ -0,0 +1,114 @@
+package com.mercedesbenz.sechub.integrationtest.api;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.mercedesbenz.sechub.integrationtest.internal.TestJSONHelper;
+
+public class AssertUserJobInfo {
+
+    public static AssertUserJobInfo assertInfo(TestSecHubJobInfoForUserListPage page) {
+        return new AssertUserJobInfo(page);
+    }
+
+    private static final Logger LOG = LoggerFactory.getLogger(AssertUserJobInfo.class);
+
+    private TestSecHubJobInfoForUserListPage listPage;
+
+    private AssertUserJobInfo(TestSecHubJobInfoForUserListPage listPage) {
+        assertNotNull(listPage);
+        this.listPage = listPage;
+    }
+
+    public AssertUserJobInfoForJob hasJobInfoFor(UUID jobUUID) {
+        return new AssertUserJobInfoForJob(jobUUID);
+    }
+
+    public class AssertUserJobInfoForJob {
+
+        private TestSecHubJobInfoForUser info;
+
+        private AssertUserJobInfoForJob(UUID jobUUID) {
+            for (TestSecHubJobInfoForUser info : listPage.getContent()) {
+                if (jobUUID.equals(info.jobUUID)) {
+                    this.info = info;
+                    break;
+                }
+            }
+            if (this.info == null) {
+                dumpInfoAndfailWith("Did not contain a job with uud:" + jobUUID);
```

Comment:
Just a small typo here.
```suggestion
                dumpInfoAndfailWith("Did not contain a job with uuid:" + jobUUID);
```